src/api/app/assets/stylesheets/webui/colors.scss

```diff
@@ -1,8 +1,16 @@
-.bg-gray-800 {
-  background-color: $gray-800;
-}
-
 .text-bg-dismissed {
   color: $white;
   background-color: $gray-500;
 }
+
+$themed-dark-nav-bg: #121212;
+
+:root {
+  --navbar-themed-bg: #{$gray-800};
+  --breadcrumbs-item-text-color: #{$gray-700};
+}
+
+@include color-mode(dark, true) {
+  --navbar-themed-bg: #{$themed-dark-nav-bg};
+  --breadcrumbs-item-text-color: #{$gray-400};
+}
```

Comment:
Don't want to have variables assigned without some kind of a parent (even if it's `:root`)
```suggestion

  @include color-mode(dark, true) {
    --navbar-themed-bg: #{$themed-dark-nav-bg};
    --breadcrumbs-item-text-color: #{$gray-400};
  }
}
```

Reply:
Did you try that on your side? 
I did it and the navbar stopped having the proper background in the dark mode, it defaults to the light one.
![imagen](https://github.com/openSUSE/open-build-service/assets/11314634/06bbd331-68d1-4fde-ad2e-c08aafe296ea)
![imagen](https://github.com/openSUSE/open-build-service/assets/11314634/58e1cbce-497c-485e-b617-2437cdcd2d21)

Can it be that the `@include color-mode(dark, true)` part does not work there?

Reply:
I think I saw it somewhere... nevermind, even without the `true` parameter it doesn't work...
I'm not sure what to do TBH

Reply:
Hm, guess you will have to invert it then and place `:root` on the inside of `@include`  to encapsulate those variables